src/jvm/test/org/jetbrains/kotlinx/lincheck_test/runner/ParallelThreadsRunnerExceptionTest.kt

```diff
@@ -152,6 +153,33 @@ class ParallelThreadsRunnerExceptionTest {
     }
 }
 
+class ParallelThreadExecutionExceptionsTest {
```

Comment:
Please move this test to a separate file and update the test name. You also do not need to mock anything here.